src/axom/quest/Shaper.hpp

```diff
@@ -112,6 +155,8 @@ class Shaper
   mint::Mesh* m_surfaceMesh {nullptr};
 
   int m_samplesPerKnotSpan {25};
```

Comment:
similar comment about using named constants vs. magic numbers here and elsewhere. It would be easier to document the rationale for choices made.

Reply:
25 is just a default value that seemed somewhat reasonable and didn't require the user to supply a value if they didn't care to. There's no other rationale for it.

Reply:
I agree that we can convert it to a `DEFAULT_SAMPLES_PER_KNOTSPAN` `constexpr` at the top of the file/class to make it somewhat less magical

Reply:
That's fine. But I still think it would be a good idea to use named constants for this sort of thing, especially when used in multiple places in the code.

Reply:
I took a stab at putting some of these default values as static constexpr members of the class and initializing using those values. For Shaper, there were some unexpected linking problems where it was as if the compiler ignored the constexpr part of the declaration and it wanted a variable in the cpp file. Weird.